### ISSUE TITLE
Korean command prompt not compatible with IS.

### ISSUE THREAD

**Issue by Illegal-Services** [Illegal-Services/Illegal_Services#10]
* v5.9.0.0 behaviour:
> Codepage: 65001
> Encoding: Supposed to be UTF-8

![v5 9 0 0](https://user-images.githubusercontent.com/62464560/153757400-6fb05e37-a95d-4472-bb8b-7c9f34240065.png)

* v5.8.0.0 behaviour:
> Codepage: Unknown
> Encoding: Supposed to be 1151

![v5 8 0 0](https://user-images.githubusercontent.com/62464560/153757401-29eb7186-2768-47fc-9b0e-84c374a17c5b.png)

**Comment by SoftieIsVibing** [comment]
is there anyway to fix?

**Comment by Illegal-Services** [comment]
yes but I don't really wanna tell you how as I don't want to be responsible for potentially unwanted behavior on your computer.

**Comment by Illegal-Services** [comment]
i suggest you to use IS website instead of the application.
https://illegal-services.github.io/Illegal_Services/Bookmarks%20Toolbar/Illegal%20Services/index.html

**Comment by SoftieIsVibing** [comment]
Okay thanks
